.github/scripts/reviewers_add.js

```diff
@@ -110,7 +106,6 @@ module.exports = async ({ github, context }) => {
   }
 
   if (labels.includes("vim-patch")) {
-    reviewers.add("seandewar");
```

Comment:
🫡

Reply:
🫡

Reply:
:saluting_face: 

Apologies for how long it's been! I expect to return eventually, but still not sure when...
Feel free to remove me as a reviewer in the meantime though. :hugs:

Reply:
You'll be welcome whenever 🤗  Take care, and drop by in chat sometime!